picard/formats/asf.py

```diff
@@ -186,8 +186,11 @@ def _save(self, filename, metadata):
         log.debug("Saving file %r", filename)
         file = ASF(encode_filename(filename))
 
-        if config.setting['clear_existing_tags']:
-            file.tags.clear()
+        if config.setting["clear_existing_tags"]:
```

Comment:
> Hmmm, what about going the pythonic way, try a method, and if it fails try another one ?

I am not sure if this is good in this case. The method might exist, but have different behavior leading to hard to find bugs. Actually if it is format specific I would leave it inside the formats for best control over the behavior.